common/app/model/dotcomrendering/pageElements/TextCleaner.scala

```diff
@@ -37,14 +60,25 @@ object Cleaners {
       case other => other
     })
   }
+
+  def split(html: String): List[(String, String)] = {
+    Jsoup
+      .parseBodyFragment(html)
+      .body()
+      .children()
+      .asScala
+      .toList
+      .map(el => (el.tagName, el.outerHtml))
+  }
+
 }
 
 object TagLinker {
 
   def link(tag: Tag, edition: Edition): String = {
     val href = LinkTo(tag.metadata.url, edition)
 
-    s"""<a href=$href data-component="auto-linked-tag">${tag.name}</a>"""
+    s"""<a href="$href" data-component="auto-linked-tag">${tag.name}</a>"""
```

Comment:
Unrelated to this change, but seemed worth fixing (although I couldn't reproduce a problem it caused)